### PR TITLE
0xsi/markdown syntax highlight

### PR DESCRIPTION
**Pull request by Simon-Busch**
- Added  **gatsby-remark-vscode** plugin to style the code blocks
- Updated Gastby config accordingly
- Updated base.css to style the blocks

New style:
Capture d’écran 2022-05-03 à 14 34 19

NB: 

⚠️ There are some code block with missing syntax in MD which cause the block to render the wrong way.
File : 2022-04-backed.md
Indentation seems to be broken as well in some blocks
 ( 
Capture d’écran 2022-05-03 à 14 31 24

### CODE REVIEW

**Review by captainmangoC4 — APPROVED** [review]
Looks good!

**Comment by joshuashort** [comment]
Hold on this a bit so @CloudEllie and team can adjust to the changes to issue bodies.

(there are some other data problems in reports/issues that can be addressed as well)

File: _data/reports/2021-09-defiprotocol.md [hunk 1]
```diff
@@ -989,7 +989,7 @@ _Submitted by WatchPug_
 
 <https://github.com/code-423n4/2021-09-defiProtocol/blob/main/contracts/contracts/Auction.sol#L143>
 
-```solidity=140
```
Comment (joshuashort) [inline _data/reports/2021-09-defiprotocol.md:992]: The semantics of some of these are a little unclear -- the blocks sometimes refer to lines outside of the (optional?) mention above them.

**Comment by Simon-Busch** [comment]
> Hold on this a bit so @CloudEllie and team can adjust to the changes to issue bodies.
> 
> (there are some other data problems in reports/issues that can be addressed as well)

Does it means that @CloudEllie  's team will adapt the markdown contests' report ? 
=> Add ```solidity where it's missing and correct indentation ? 

I don't mind doing it that will be quite quick ! 

I believe it makes sense as well to add in the same PR a mention in the text area for wardens to know the syntax they have to use, do you agree @tealtigrrr  ?

**Comment by Simon-Busch** [comment]
Merging with approval from @joshuashort

**Pull request merged**